client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts

```diff
@@ -207,7 +207,7 @@ export class ListOfSpeakersContentComponent extends BaseMeetingComponent impleme
         if (speakerToDelete && (await this.promptService.open(title))) {
             await this.speakerRepo.delete(speakerToDelete.id);
             this.filterNonAvailableUsers();
-            this.interactionService.kickUsers([speaker.user], `Removed from the list of speakers`);
+            this.interactionService.kickUsers([speakerToDelete.user], `Removed from the list of speakers`);
```

Comment:
This looks like a bug which should be fixed regardless, right?

Reply:
Absolutely, I'll make an issue

Reply:
-> #2232